lib/rubocop/cop/rails/find_each.rb

```diff
@@ -35,7 +35,7 @@ class FindEach < Base
 
         SCOPE_METHODS = %i[
           all eager_load includes joins left_joins left_outer_joins not preload
-          references unscoped where
+          references unscoped where or
```

Comment:
Can you update to order by ASCII position?

```diff
+          all eager_load includes joins left_joins left_outer_joins not or preload
+          references unscoped where
-          all eager_load includes joins left_joins left_outer_joins not preload
-          references unscoped where 
```

Reply:
I fixed it!